vagrant/provision-liquid.sh

```diff
@@ -15,6 +15,9 @@ fi
 
 cp examples/liquid.ini .
 
+sudo apt install python3-venv python3-pip
```

Comment:
Please use `apt-get` instead of `apt`. Also, add `-qy` as arguments, so it won't prompt the user for confirmation.